### DISCUSSION TITLE
Assistance Needed with Student Status Verification for GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by UmairAhmed406** [community/community#119690] [GitHub Education]
### Hi everyone,

Dear GitHub Support Team,

I am reaching out regarding an issue I’ve encountered while attempting to verify my student status for access to the Student Developer Pack. I have attempted verification using my student credentials from my university, the Institute of Space Technology, Islamabad.

The verification process requires a student email address, and I’ve faced difficulties due to this requirement. I have two email addresses provided by my university for official communication:

	1.	An email address primarily used for Office 365 and MS Teams, which has been successfully utilized for verification on other platforms.
	2.	Another email address that historically functioned but is currently inaccessible, along with the university portal for semester results, due to a cyber attack.

I have attached my Student Card in PDF format and last semester’s results to support my verification. However, I am unable to receive any correspondence through either email address.

I kindly request your assistance in finding an alternative method or solution to verify my student status and gain access to the Student

**Comment by samus-aran** [comment]
Hi @UmairAhmed406 

I'm not sure what you are asking. You have academic emails you can use to verify or not? If you have an email associated with Institute of Space Technology, Islamabad then use this in the [application form](https://education.github.com/discount_requests/application). We do not process manual reviews, all applications must go through the online platform.

**Reply by UmairAhmed406** [reply]
I do have an email address, but I can’t access it right now. When I am asked to select my university and I do, it asks me to go to this address (mail.ist.edu.pk) and sign in to my email. However, the address provided don’t even work. 
![IMG_5151](https://github.com/community/community/assets/54098606/bb31089c-6351-4fb8-a35a-18e66159873e)
There was a cyber attack on our university platform, which affected the servers, and we are unable to log in to our email. This issue is not only affecting me but also our faculty. My question right now is, is there any other way I can verify?
